tubular/scripts/frontend_deploy.py

```diff
@@ -22,6 +22,10 @@
 
 
 @click.command("frontend_deploy")
+@click.option(
+    '--common-config-file',
+    help='File from which common configuration variables are read.',
+)
```

Comment:
[inform/context] MFEs will need to start supplying the `--common-config-file` when running this script to provide the path to the common frontend config. Currently assumes `DATADOG_API_KEY` can be exposed in this common frontend config.